src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,589 @@
+    """
+    _unhandled[s.split(':')[0]] += 1
+    logger.warning('unhandled identifier: %s', s)
+    return
+
+
+def get_processed_intact_df() -> pd.DataFrame:
+    """Load, filter and rename intact dataframe."""
+    path = ensure_path(prefix=MODULE_NAME, url=URL)
+    logger.info('reading IntAct from %s', path)
+    with ZipFile(path) as zip_file:
+        with zip_file.open('intact.txt') as file:
+            df = pd.read_csv(file, sep='\t', usecols=COLUMNS, na_values={'-'})
+
+    df.dropna(inplace=True)
+
+    # Omit certain interaction types
+    df = df[~df['Interaction type(s)'].isin(INTACT_OMIT_INTERACTIONS)]
+
+    df['#ID(s) interactor A'] = df['#ID(s) interactor A'].map(_process_interactor)
+    df['ID(s) interactor B'] = df['ID(s) interactor B'].map(_process_interactor)
+
+    # filter for pubmed
+    logger.info('mapping provenance')
+    df['Publication Identifier(s)'] = df['Publication Identifier(s)'].map(_process_pmid)
+
+    return df
+
+
+def get_bel() -> BELGraph:
+    """Get BEL graph."""
+    df = get_processed_intact_df()
+    graph = BELGraph(name=MODULE_NAME, version=VERSION)
+    it = tqdm(df[COLUMNS].values, total=len(df.index), desc=f'mapping {MODULE_NAME}', unit_scale=True)
+    for source_uniprot_id, target_uniprot_id, relation, pubmed_id, detection_method, source_db, confidence in it:
+        if pd.isna(source_uniprot_id) or pd.isna(target_uniprot_id):
+            continue
+        _add_my_row(
+            graph,
+            relation=relation,
+            source_uniprot_id=source_uniprot_id,
+            target_uniprot_id=target_uniprot_id,
+            pubmed_id=pubmed_id,
+            int_detection_method=detection_method,
+            source_database=source_db,
+            confidence=confidence,
+        )
+    return graph
+
+
+def _add_my_row(
+    graph: BELGraph,
+    relation: str,
+    source_uniprot_id: str,
+    target_uniprot_id: str,
+    pubmed_id: str,
+    int_detection_method: str,
+    source_database: str,
+    confidence: str,
+) -> None:  # noqa:C901
+    """Add for every pubmed ID an edge with information about relationship type, source and target.
+
+    :param graph: graph to add edges to
+    :param relation: row value of column relation
+    :param source_uniprot_id: row value of column source
+    :param target_uniprot_id: row value of column target
+    :param pubmed_id: row value of column pubmed_id
+    :param int_detection_method: row value of column interaction detection method
+    :param confidence: row value of confidence score column
+    :return: None
+    """
+    if pubmed_id is None:
+        return
+
+    annotations = {
+        'psi-mi': relation,
+        'intact-detection': int_detection_method,
+        'intact-source': source_database,
+        'intact-confidence': confidence,
+    }
+    # map double spaces to single spaces in relation string
+    relation = ' '.join(relation.split())  # FIXME how often does this happen? can you tweet Intact with the number?
```

Comment:
check FIXME

Reply:
I had put a comment there : 

# I only found it in 'psi-mi:"MI:1237"(proline isomerization reaction)', nowhere else